File: public/usage-examples/graphics/draw_circle_on_window/draw_circle_on_window-1-random-color-circle.cpp
#include "splashkit.h"

int main()
{
    window myWindow = open_window("Draw Circle on Window", 800, 600);
    clear_screen();
    
    for (int i = 0; i < 50; i++)
    {
        double x = rand() % 800;
        double y = rand() % 600;
        double radius = rand() % 50;

        color randomColor = rgb_color(rand() % 255, rand() % 255, rand() % 255);

        draw_circle_on_window(myWindow, randomColor, x, y, radius);
    }
    
    refresh_screen();
    delay(4000);
    close_all_windows();

    return 0;
}

File: public/usage-examples/graphics/draw_circle_on_window/draw_circle_on_window-1-random-color-circle.py
from splashkit import *
import random

my_window = open_window("Draw Circle on Window", 800, 600)
clear_screen(color_white())

for _ in range(50):
    x = random.randint(0, 800)
    y = random.randint(0, 600)
    radius = random.randint(0, 50)

    random_color = rgb_color(
        random.randint(0, 255), random.randint(0, 255), random.randint(0, 255)   
    )

    draw_circle_on_window(my_window, random_color, x, y, radius)

refresh_screen()
delay(4000)
close_all_windows()
